client/src/global.css

```diff
@@ -561,3 +561,7 @@ TODO: override default styles
   color: blue;
   text-decoration: underline;
 }
+
+.rr-average-rating {
+  font-size: 50px;
```

Comment:
Can you convert this to rem. 50px is about 3.8rem with our current base font size.

Reply:
yes ill do that now